app/Services/PackageLoaderService.php

```diff
@@ -0,0 +1,33 @@
+<?php
+namespace App\Services;
+/**
+ * i don't know, need it or not..
+ * use Illuminate\Contracts\Container\Container;
+ */
+class PackageLoaderService
+{
+    function __construct()
+    {
+        //I don't know that there should be...
+    }
+    public static function listPackages()
```

Comment:
I think this is unnecessary.
We can just iterate through `packages.` and set the array for menu. In this case we will get rid of `if` checks.